reference-implementation/lib/transform-stream.js

```diff
@@ -334,7 +338,7 @@ class TransformStreamDefaultSource {
   }
 
   pull() {
-    // console.log('TransformStreamDefaultSource.pull()');
+    verbose('TransformStreamDefaultSource.pull()');
```

Comment:
Can we name this TransformStreamDefaultSource.prototype.pull()?

Reply:
Done.